TimberAPI/Dependency/IDependency.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Bindito.Core;
```

Comment:
It might make sense to use `git mv` to retain file history. After all it's renamed and edited, not really added.

(Same goes for a few other files. Not gonna spam you with the same remark everywhere. :))

Reply:
https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history
It looks like I can't retain history if rename the file, only if I git mv the folder (but in this case I wasn't changing the folder) :(

Reply:
Right. It might make sense to split renaming and editing into two separate commits. Maybe we just live with it in this instance and keep it in mind for the future. :)